src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -28,24 +28,36 @@ abstract class Vendedor {
   fun otrasCertificaciones() = certificaciones.count { !it.esDeProducto }
 
   fun puntajeCertificaciones() = certificaciones.sumBy { c -> c.puntaje }
+
+  abstract fun esInfluyente() : Boolean
 }
 
 // En los parámetros, es obligatorio poner el tipo
 class VendedorFijo(val ciudadOrigen: Ciudad) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudad == ciudadOrigen
   }
+
+  override fun esInfluyente(): Boolean = false
+
 }
 
 // A este tipo de List no se le pueden agregar elementos una vez definida
 class Viajante(val provinciasHabilitadas: List<Provincia>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return provinciasHabilitadas.contains(ciudad.provincia)
   }
+  override fun esInfluyente(): Boolean =
+    provinciasHabilitadas.sumBy { p -> p.poblacion } >= 10000000
 }
 
 class ComercioCorresponsal(val ciudades: List<Ciudad>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudades.contains(ciudad)
   }
+
+// REVISAR
+  override fun esInfluyente(): Boolean =
+  (ciudades.size >= 5) or (ciudades.map { c -> c.provincia }.toSet().size >= 3)
+
```

Comment:
Otra solución para esto:
* Delegar el cálculo a otro método. 
* Los parentesis no son necesarios.
* Si definís a las consultas (los métodos que devuelven) con `=` no es necesario explicitar el tipo de dato.
* `ciudades.map { ciudad -> ciudad.provincia }` es equivalente a `ciudades.map { it.provincia }`

```suggestion
  fun cantidadDeProvincias() = ciudades.map { it.provincia }.toSet().size
  override fun esInfluyente() = ciudades.size >= 5 || this.cantidadDeProvincias() >= 3

```